README.md

```diff
@@ -377,6 +377,9 @@ Let us know if you have any questions!
 
 - No. While data that is sent over POST may be more protected, the information could easily be intercepted by a third party or middleman, and Google has complete access to the data inside a Google Spreadsheet. Email is also not a very secure communication medium by default. We would recommend you invest in a secure platform and server for storing your data if this is a requirement.
 
+8. _What if my data is being collected or sent in the wrong order?_
```

Comment:
nit: this reads a little odd, could change to "being stored" or just have it say "sent"